README.rst

```diff
@@ -9,70 +9,89 @@ Open Zaak
+
+.. _`Zaakgericht werken in het gemeentelijk gegevenslandschap`: https://www.gemmaonline.nl/images/gemmaonline/f/f6/20190620_-_Zaakgericht_werken_in_het_Gemeentelijk_Gegevenslandschap_v101.pdf
 
-Inleiding
+
+Standaard "API's voor Zaakgericht Werken"
 =========
 
-Binnen het Nederlandse gemeentelandschap wordt zaakgericht werken nagestreefd.
-Om dit mogelijk te maken is er gegevensuitwisseling nodig. De kerngegevens van
-zaken, documenten en besluiten moeten ergens geregistreerd worden en
-opvraagbaar zijn.
+In het kader van Common Ground heeft VNG Realisatie deze standaard ontwikkeld. Daarbij zijn tegelijk met API-specificaties referentie-implementaties gerealiseerd om aan te tonen dat de specificaties in software kunnen worden geïmplementeerd. De volgende inhoudelijke API's maken onderdeel uit van de standaard:
+
+* Catalogi - voor de registratie van zaaktype-catalogi, zaaktype en alle daarbij horende typen.
+* Zaken - voor de registratie van zaken. Zaken kunnen o.a. relaties hebben met documenten, besluiten, contacten. De API biedt functionaliteit voor audit trail en archiveren.
+* Documenten - voor de registratie van informatieobjecten, hetgeen zowel documenten als andere informatiedragers zoals foto's en film kunnen zijn.
+* Besluiten - voor de registratie van besluiten die in het kader van zaakgericht werken worden genomen.
+* Klantinteracties - deze API ondersteunt de registratie van Contactmomenten en Verzoeken. Verzoeken hebben een relatie met producten en diensten uit de Producten Diensten Catalogus (PDC) en kunnen leiden tot zaken. Deze API wordt op een later moment toegevoegd aan OpenZaak.
 
-**Open Zaak** is een moderne, open source gegevens- en services-laag om
-`zaakgericht werken`_ te ondersteunen zoals voorgesteld in de onderste 2 lagen
-van het `Common Ground`_ 5-lagen model.
+Daarnaast zijn er een paar generieke API's die nodig om gebruik te maken van deze API's:
 
-De gegevens worden ontsloten middels een `gestandaardiseerde set VNG API's`_,
-te weten:
+* Notificaties - in Common Ground worden gegevens bij de bron geregistreerd en bijgehouden. Consumers krijgen niet vanzelf bericht als er iets is gewijzigd. Hiervoor kunnen ze een abonnement registreren bij de Notificaties API.
+* Autorisaties - via de Autorisaties API wordt de toegang van applicaties tot gegevens geregeld.
 
-* Zaken API (`Zaken API-specificatie 1.0`_)
-* Documenten API (`Documenten API-specificatie 1.0`_)
-* Catalogi API (`Catalogi API-specificatie 1.0`_)
-* Besluiten API (`Besluiten API-specificatie 1.0`_)
-* Autorisaties API (`Autorisaties API-specificatie 1.0`_)
+Productiewaardige API's
+==========
 
-De `Notificaties API`_ is nodig voor **Open Zaak** en is beschikbaar via
```

Comment:
Ik mis deze niet onbelangrijke zin. Het is nodig (niet alleen "het is er") en de verwijzing naar Open Notificaties.